microsoft-365/security/mtp/mtp-enable.md

```diff
@@ -30,6 +30,9 @@ search.appverid:
 
 Microsoft Threat Protection automatically turns on when eligible customers with the required permissions visit Microsoft 365 security center. Read this article to understand various prerequisites and how Microsoft Threat Protection is provisioned.
 
+>[!NOTE]
+>To learn about Advanced Threat Protection, see [Office 365 Advanced Threat Protection](https://docs.microsoft.com/microsoft-365/security/office-365-security/office-365-atp?view=o365-worldwide).
+
```

Comment:
Remove. Does not make sense to add a link to O365 in this very prominent section. Also:

o	“Advanced Threat Protection” is not an approved name for Office 365 ATP. In fact, there are other ATPs—Microsoft Defender ATP and Azure ATP. BTW, how do we know that searches for “ATP” are intended for Office 365 ATP?
o	While I understand this is inserted for SEO or to manage entries from organic searches for “ATP”, I feel this will read like “upsell” text that is too prominent and inserted out of context. Actually, it can be good that they landed on the MTP enablement page because MTP is a higher value superset that we want people to use vs the legacy OATP.